application/__init__.py

```diff
@@ -31,8 +31,9 @@ def create_app(config_filename=None):
     app.config.from_pyfile(config_filename)
     app.es = (Elasticsearch(app.config['ELASTICSEARCH_URL'])
               if app.config['ELASTICSEARCH_URL'] else None)
-    app.graph = (Database(app.config['NEO4J_URL']).default_graph
-                 if app.config['NEO4J_URL'] else None)
+    config.ENCRYPTED_CONNECTION = False
```

Comment:
Why not encrypted?

Reply:
Is a pain to setup for development and our database is not available remotely anyways.

Reply:
Actually nevermind, the problem seems to have been something else. I enabled it again :)